{
  "modified": "2024-06-25T13:32:08Z",
  "published": "2024-06-25T13:32:08Z",
  "schema_version": "1.5.0",
  "id": "MAL-2024-4734",
  "summary": "Malicious code in adv2099m (PyPI)",
  "details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
  "affected": [
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "adv2099m",
        "purl": "pkg:pypi/adv2099m"
      },
      "versions": [
        "1.0.0"
      ]
    }
  ],
  "credits": [
    {
      "name": "ReversingLabs",
      "type": "FINDER",
      "contact": [
        "https://www.reversinglabs.com"
      ]
    }
  ],
  "database_specific": {
    "malicious-packages-origins": [
      {
        "id": "RLMA-2024-03514",
        "import_time": "2024-06-28T02:48:12.398884134Z",
        "modified_time": "2024-06-25T13:32:08Z",
        "sha256": "9c8a27717d98562d0fcb83166a4942de47c41ac517a78fb3e16bd4d7fefe87d0",
        "source": "reversing-labs",
        "versions": [
          "1.0.0"
        ]
      }
    ]
  }
}
